monitor/census.go

```diff
@@ -158,6 +158,7 @@ func InitCensus(nodeType, nodeID, version string) {
 		success:     make(map[uint64]*segmentsAverager),
 	}
 	var err error
+	ctx := context.Background()
```

Comment:
This could be passed into `InitCensus`, but I'm guessing you created a background context here to minimize changes with the intent of adjusting context flow later on?

Reply:
Correct.  This first pass does not alter any function prototypes.   The next one does.